assets/src/edit-story/elements/video/index.js

```diff
@@ -32,6 +32,7 @@ export const defaultAttributes = {
   posterId: null,
   poster: null,
   videoId: 0,
+  isBackground: true,
```

Comment:
Why adding `isBackground` as default attributes for all elements now? And why setting it to `true` here?

My understanding was that individual elements shouldn't be concerned about it because we have `backgroundElementId`, and reducers like `setBackgroundElement` will set `isBackground` dynamically.

Reply:
Video are background true. This is so that it an element is background can be defined at creation of element.

Reply:
Hm, `isBackground` ought to be a reserved property for elements (not allowed to be explicitly set by a new element or in an element update) and the only "true" way to set an element as background is to use `setBackgroundElement`.

I think this kind of breaks the system, as `addElements` doesn't check for this attribute and thus doesn't correctly update the `page.backgroundElementId` property.

If we want to be able to do it this way, the system has to properly support it. We did settle on [using the explicit setter](https://github.com/ampproject/amp-wp/pull/3910#issuecomment-564783500) when the system was created, but that can of course be reversed.

For now, treat `isBackground` as read-only until we decide on whether we should support setting it.

Reply:
Okay not a fan of the way I did this. Removed in https://github.com/google/web-stories-wp/pull/376/commits/10e7278acaf796fa565433cd06951c5134d9dff5